docs/examples/shop-bot.js

```diff
@@ -32,14 +31,14 @@ const shippingOptions = [
 ]
 
 const replyOptions = Markup.inlineKeyboard([
-  Markup.payButton('💸 Buy'),
-  Markup.urlButton('❤️', 'http://telegraf.js.org')
-]).extra()
+  Markup.button.pay('💸 Buy'),
+  Markup.button.url('❤️', 'http://telegraf.js.org')
+])
 
 const bot = new Telegraf(process.env.BOT_TOKEN)
 bot.start(({ replyWithInvoice }) => replyWithInvoice(invoice))
 bot.command('buy', ({ replyWithInvoice }) => replyWithInvoice(invoice, replyOptions))
-bot.on('shipping_query', ({ answerShippingQuery }) => answerShippingQuery(true, shippingOptions))
+bot.on('shipping_query', ({ answerShippingQuery }) => answerShippingQuery(true, shippingOptions, undefined))
```

Comment:
We may need to redefine that function's signature. It would be nice if `undefined` is not required here.

Reply:
BY adding `true` and `false` overloads?

Reply:
How are you going to make `true` and `false` overloads for `string | undefined`?

Reply:
Nvm, you mean overloads for `ok`. Yes, that makes a lot of sense.